src/Blazored.Modal/BlazoredModal.razor

```diff
@@ -1,30 +1,8 @@
-﻿@using Microsoft.AspNetCore.Components.Web
-
-<div class="blazored-modal-container @ComponentPosition @(IsVisible ? "blazored-modal-active" : string.Empty)">
-
-    <div class="blazored-modal-overlay" @onclick="HandleBackgroundClick"></div>
-
-    <div class="blazored-modal-wrapper">
-        <div class="@ComponentStyle">
-            @if (!ComponentHideHeader)
-            {
-                <div class="blazored-modal-header">
-                    <h3 class="blazored-modal-title">@Title</h3>
-                    @if (!ComponentHideCloseButton)
-                    {
-                        <button type="button" class="blazored-modal-close" @onclick="@(() => ModalService.Cancel())">
-                            <span>&times;</span>
-                        </button>
-                    }
-                </div>
-            }
-            <div class="blazored-modal-content">
-                <CascadingValue Value="this">
-                    <CascadingValue Value="@Parameters">
-                        @Content
-                    </CascadingValue>
-                </CascadingValue>
-            </div>
-        </div>
-    </div>
-</div>
+﻿<CascadingValue Value="this">
+    <CascadingValue Value="GlobalModalOptions">
+        @foreach (var modalInstance in Modals)

```

Comment:
I think this should be
```suggestion
        @foreach (var modalReference in Modals)

```
to avoid confusion.

Reply:
Maybe it would be better to side step this altogether and just go with `modal` as the loop variable?
```suggestion
        @foreach (var modal in Modals)

```

Reply:
I agree, that is even better.